src/components/CreateProfile.js

```diff
@@ -254,6 +258,61 @@ const CreateProfile = ({
         </Box>
       )}
 
+      {tab === 'blockstack' && (
```

Comment:
For a first version, let's not modify the `CreateProfile` page, and just use the link on the `SignIn`page or similar.  All modifications to this file can be removed.

Reply:
Changes to CreateProfile have been reverted in https://github.com/opencollective/opencollective-frontend/pull/1980